.gitignore

```diff
@@ -0,0 +1,75 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
So https://www.gitignore.io/ actually gives me more stuff.  Unless somebody feels strongly, I'm going to leave it be.